pint/definitions.py

```diff
@@ -8,17 +8,79 @@
     :license: BSD, see LICENSE for more details.
 """
 
+from collections import namedtuple
+
 from .converters import OffsetConverter, ScaleConverter
 from .errors import DefinitionSyntaxError
 from .util import ParserHelper, UnitsContainer, _is_dim
 
 
+class ParsedDefinition(
+    namedtuple("ParsedDefinition", "name symbol aliases value rhs_parts")
+):
+    """Splits a definition into the constitutive parts.
+
+    A definition is given as a string with equalities in a single line.
+
+        ---------------> rhs
+    a = b = c = d = e
+    |   |   |   -------> aliases (optional)
+    |   |   |
+    |   |   -----------> symbol (use "_" to
+    |   |
+    |   ---------------> value
+    |
+    -------------------> name
+
+    Attributes
+    ----------
+    name : str
+    value : str
+    symbol : str or None
+    aliases : tuple of str
+    rhs : tuple of str
+    """
+
+    @classmethod
+    def from_string(cls, definition):
+        name, definition = definition.split("=", 1)
+        name = name.strip()
+
+        rhs_parts = tuple(res.strip() for res in definition.split("="))
+
+        value, aliases = rhs_parts[0], tuple([x for x in rhs_parts[1:] if x != ""])
```

Comment:
```suggestion
        value, aliases = rhs_parts[0], tuple(x for x in rhs_parts[1:] if x != "")
```
you can just use the generator expression instead of a list comprehension converted to a tuple

Reply:
agreed. I will fix this.